### ISSUE TITLE
Improve the steps in contributing to the project

### ISSUE THREAD

**Issue by Ginohmk** [conda-forge/conda-forge.github.io#1727]
### Where should the content be added?

https://conda-forge.org/docs/user/contributing.html

### What should be added?

- I suggest that the steps in contributing to a project do not properly stand out, especially for a beginner. I can write more easy steps to make it better and easy to contribute to the project.
- Also I noticed some terms are outdated like that of the branch name as ``master`` instead of  ``main``

### Additional information

.

**Comment by Ginohmk** [comment]
@kathatherine @beckermr  if this issue is good enough, I will like to commence work on fixing it. 

Please I need your view on this. Thank you.

**Comment by kathatherine** [comment]
@Ginohmk I think we have a PR out there that renames all instances of "master" to "main", so don't worry about that. I would agree that it would be great to have all external links open in a new tab. Just the external links, though. For all links within the conda-forge docs, it would be better to let the user choose.

**Comment by Ginohmk** [comment]
@kathatherine  thanks for your reply. From what you suggested. Can I work on making just the **external links**  open in a new tab ?

**Comment by kathatherine** [comment]
@Ginohmk Yes, yes. You are welcome to work on it. I don't have the permissions to assign you, but with any issue, you can just say, "I'm going to work on this" so everyone's aware and then get started.

**Comment by Ginohmk** [comment]
Much Thanks @kathatherine 

I am going to work on this issue

**Comment by kathatherine** [comment]
@Ginohmk Hmm, good to know. Thank you for your research on this. It does seem rather painful to implement. Adding javascript to the conf.py would affect every link, I'm assuming, and we don't want that and I'd also like to avoid adding raw HTML everywhere in the docs, as it would really clutter things up. I think it would be best to leave the issue for now.

**Comment by Ginohmk** [comment]
You are welcome @kathatherine . I thought of the complications attached to implementing it now with javascript, and it's something that would not seat right with the previous code structure. Hopefully, in the future, it would be an added feature that can be implemented easily :)
And thanks for your quick reply. I will scan through the issues available and see if I can work on something else. Thank you.
